ksql-engine/src/main/java/io/confluent/ksql/internal/KsqlEngineMetrics.java

```diff
@@ -49,14 +56,21 @@
 
 
   private final KsqlEngine ksqlEngine;
+  private final Metrics metrics;
 
   public KsqlEngineMetrics(final String metricGroupPrefix, final KsqlEngine ksqlEngine) {
+    this(metricGroupPrefix, ksqlEngine, MetricCollectors.getMetrics());
+  }
+
+  public KsqlEngineMetrics(final String metricGroupPrefix, final KsqlEngine ksqlEngine,
```

Comment:
nit: make package local - the idea is that this constructor should only be used from tests.

Reply:
Made it package local as part of the challenge!